test/core/surface/sequential_connectivity_test.cc

```diff
@@ -100,12 +100,14 @@ static void run_test(const test_fixture* fixture) {
 
   while (grpc_completion_queue_next(server_cq,
                                     gpr_inf_future(GPR_CLOCK_REALTIME), nullptr)
-             .type != GRPC_QUEUE_SHUTDOWN)
+             .type != GRPC_QUEUE_SHUTDOWN) {
     ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done